code/game/gamemodes/miniantags/revenant/revenant_abilities.dm

```diff
@@ -114,8 +114,8 @@
 				charge_counter = charge_max
 				return
 			log_say("RevenantTransmit: [key_name(user)]->[key_name(M)] : [msg]")
-			user << "<span class='revennotice'><b>You transmit to [M]:</b> [msg]</span>"
-			M << "<span class='revennotice'><b>An alien voice resonates from all around...</b></span><i> [msg]</I>"
```

Comment:
The msg bit is its own part. I can change it to use a different span but there wasn't a revenant span with italics.